hera_sim/sigchain.py

```diff
@@ -488,6 +500,188 @@ def __call__(self, freqs, autovis, **kwargs):
         return crosstalk_spectrum
 
 
+class OverAirCrossCoupling(Crosstalk):
+    r"""Crosstalk model based on the mechanism described in HERA Memo 104.
+
+    This model describes first-order coupling between a visibility :math:`V_{ij}`
+    and the autocorrelations for each antenna involved. Physically, it is modeled
+    as the signal from one antenna traveling to the receiverator, then being
+    broadcast to the other antenna. Under this model, the cross-coupling component
+    :math:`V_{ij}^{\rm cc}` can be described via
+
+    .. math::
+
+        V_{ij}^{\rm cc} = \epsilon_{ij}^* V_{ii} + \epsilon_{ji} V_{jj},
+
+    where the reflection coefficient :math:`\\epsilon_{ij}` is modeled as
+
+    .. math::
+
+        \epsilon_{ij} = A_i \exp \bigl[2\pi i\nu(\tau_{i,{\rm cable}} +
+        \tau_{X \rightarrow j} ) \bigr].
+
+    Here, :math:`X` denotes the position of the receiverator (or rather, where the
+    excess signal is radiated from), and the indices :math:`i,j` refer to antennas.
+    So, :math:`\tau_{i,{\rm cable}}` is the delay from the signal traveling down
+    the cable from antenna :math:`i` to the receiverator, and :math:`\tau_{X
+    \rightarrow j}` denotes the delay from the signal traveling over-the-air from
+    the receiverator to antenna :math:`j`. As usual, :math:`A_i` is the amplitude
+    of the reflection coefficient. Here, the amplitude is described by three free
+    parameters, :math:`a, \vec{r}_X, \beta`:
+
+    .. math::
+       A_i = a |\vec{r}_i - \vec{r}_X|^\beta.
+
+    :math:`a` is a base amplitude, :math:`\vec{r}_X` is the receiverator position,
+    and :math:`\beta` describes how quickly the amplitude falls off with distance
+    from the receiverator, and is typically taken to be negative. For more details,
+    refer to HERA Memo 104 for more details:
+
+    http://reionization.org/manual_uploads/HERA104_Crosstalk_Physical_Model.html
+
+    Parameters
+    ----------
+    emitter_pos
+        Receiverator position, in meters, in local ENU coordinates.
+    cable_delays
+        Mapping from antenna numbers to cable delays, in nanoseconds.
+    base_amp
+        Base amplitude of reflection coefficient. If `amp_slope` is set to 0, then
+        this is the amplitude of all of the reflection coefficients.
+    amp_slope
+        Power-law index describing how rapidly the reflection coefficient decays
+        with distance from the receiverator.
+    amp_decay_base
+        Logarithmic base to use when generating the additional peaks in the
+        cross-coupling spectrum.
+    n_copies
+        Number of peaks to include in the cross-coupling spectrum.
```

Comment:
Clarify that this is the number of peaks at both positive and negative delays separately.

Reply:
Done, let me know if it's okay.

Reply:
Did you push the change?

Reply:
Oh, sorry, I guess I had a typo in the branch I pushed it to. Should be fixed now.

Reply:
looks good to me.